index.js

```diff
@@ -1,8 +1,9 @@
 import 'dotenv/config'
 import * as fs from 'node:fs'
-import chalk from 'chalk'
-import closeWithGrace from 'close-with-grace'
+
+import { installGlobals } from '@remix-run/node'
 import sourceMapSupport from 'source-map-support'
+installGlobals({ nativeFetch: true })
```

Comment:
This should not be necessary with Node 20. Why is this added?

Reply:
You're right, I just remembered that